### PR TITLE
Move NBO edit to functional syntax to allow easier diffing #78 

### PR DESCRIPTION
**Pull request by DitchingIt**
I am puzzled about the other commits and reversions:

1. The other day I updated nbo-idranges.owl and this was merged, but GitHug desktop got me to merge branch 'master' just now which turned into a commit here, which I reverted on GitHub desktop, and has turned into another commit here: I don't want it to undo the nbo-idranges change made the other day. Will it?
2. I reverted the Ditch Functional Syntax file on GitHub desktop, but it and the change it reverted have turned into commits here.

### CODE REVIEW

File: src/ontology/nbo-idranges.owl [hunk 1]
```diff
@@ -59,7 +59,3 @@ EquivalentTo: xsd:integer[> 50000 , <= 59999]
 Datatype: idrange: 12
 Annotations: allocatedto: "Nicole Vasilevsky"
 EquivalentTo: xsd:integer[> 60000 , <= 69999]
-
-Datatype: idrange: 13
-Annotations: allocatedto: "Ditch Townsend"
-EquivalentTo: xsd:integer[> 70000 , <= 79999]
```
Comment (matentzn) [inline src/ontology/nbo-idranges.owl:65]: You should undo this change. You can easily do this by running:

```
git checkout master -- src/ontology/nbo-idranges.owl
```

In your terminal and create a new commit.

**Review by matentzn — CHANGES_REQUESTED** [review]
The edit file looks good, but please undo the change to idranges!

File: src/ontology/nbo-idranges.owl (hunk 1 above)
Comment (DitchingIt) [reply src/ontology/nbo-idranges.owl:65]: Totally lost. Sorry.

1. Do I cancel the pull request and start again? I haven't so far:
2. I had to google what 'terminal' was. I was able to pull up a command line thing and when I copied your instruction and right clicked in the command line (which I discovered by trial and error was the way to paste), I got an error (see screenshot).
Screenshot_20230212_112147
3. In due course, would I make the new commit from the Terminal command line, or if it works, what will appear by running your instruction?
4. Why did GitHub desktop make me merge a branch master when the update was already done and merged several days ago? I reverted it because I thought it was an error.
5. Why can't I just go to GitHug desktop and revert the revert?
6. Why did I need to use the NBO-edit filename this time, whereas my own filename worked with the idnames?
7. Was I right to revert yesterday's Ditch functional syntax commit? I just guessed I needed to do that - maybe I didn't but it had no consequences anyway?

I appreciate that you want me to learn things, but would it help more for you just to edit this pull request yourself as there seem to be too many issues I need to understand before I can do this safely myself.

Comment (matentzn) [reply src/ontology/nbo-idranges.owl:65]: > Why can't I just go to GitHug desktop and revert the revert?

You can do that too! Actually, it is probably better.

I think you can call the terminal on windows by clicking, in GitHub Desktop, on the "repository" menu ("terminal" submenu). also, you will have to navigate in the terminal to the repository.

I recommend doing this tutorial: https://oboacademy.github.io/obook/tutorial/intro-cli-1/

Comment (DitchingIt) [reply src/ontology/nbo-idranges.owl]: I reverted the revert. I think we can move on.

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
Excellent job @DitchingIt!

**Comment by DitchingIt** [comment]
Thanks. I'll keep working through those tutorials too.
